modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -758,15 +758,15 @@ public void prepareRemoval(KillEvent killEvent)
     @Override
     public void postRemoval(KillEvent ce)
     {
-        _log.info("CellAdapter : prepareRemoval : waiting for gate to open");
+        logger.info("CellAdapter : prepareRemoval : waiting for gate to open");
```

Comment:
and as we not live in an ideal word, should we continue changing the renaming together with the changing of the string-concatenations or should we rather focus on re-re-naming the "logger" to "LOGGER" or "LOG"? or should we rather just change the string-concatenations? tomorrow Marissa and me are going to focus on the other exercise, but we wanted to continue with this one, as far as we got time for it.